Code/Framework/AzToolsFramework/AzToolsFramework/UI/DPEDebugViewer/DPEDebugModel.h

```diff
@@ -33,36 +33,50 @@ namespace AzToolsFramework
             PropertyEditorNode
         };
 
-        DPEModelNode(NodeType nodeType, size_t domValueIndex, QObject* theModel);
+        DPEModelNode(QObject* theModel);
         ~DPEModelNode();
 
-        int GetChildCount() const;
+        int GetRowChildCount() const;
         int GetColumnChildCount() const;
-        int GetMaxChildColumns() const;
         DPEModelNode* GetParentNode() const;
+        DPEModelNode* GetParentRowNode() const;
+
         QVariant GetData(int role) const;
         bool SetData(int role, const QVariant& value);
         Qt::ItemFlags GetFlags() const;
-        DPEModelNode* GetChildNode(int childIndex);
-        DPEModelNode* GetColumnNode(int columnIndex);
+        DPEModelNode* GetRowChild(int childIndex);
+        DPEModelNode* GetColumnChild(int columnIndex);
+
+        DPEModelNode* GetChildFromDomIndex(size_t domIndex);
 
         int RowOfChild(DPEModelNode* const childNode) const;
-        void Populate(const AZ::Dom::Value& domVal);
+        int ColumnOfChild(DPEModelNode* const childNode) const;
+
+        void SetValue(const AZ::Dom::Value& domVal, bool notifyView);
+        AZ::Dom::Value GetValueFromDom() const;
+
+        size_t GetDomValueIndex() const
+        {
+            return m_domValueIndex;
+        }
+
+        bool isColumn() const
```

Comment:
nit: this should be upper camel case

Reply:
This is not a `nit`, but actually in the Coding Standard: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#function-names

Reply:
yep, missed one. Fixed now.